website/content/en/preview/tasks/amitasks.md

```diff
@@ -47,24 +47,42 @@ However, with this comes the risk that the new AMI could break or degrade your w
+* Task 3 (Disruption budgets): This task can be used as a way of mitigating the scope of impact if a new AMI causes problems with your workloads. With Disruption budgets you can slow the pace of upgrades to nodes with new AMIs or make sure that upgrades only happen during selected dates and times (using `schedule`). This doesn't prevent a bad AMI from being deployed, but it allows you to control when nodes are upgraded, and gives you more time respond to rollout issues.
 
 ## Tasks
 
-The following tasks let you have an on impact Karpenter’s behavior as it relates to how nodes are created and AMIs are consumed.
+The following tasks let you have an impact on Karpenter’s behavior as it relates to how nodes are created and AMIs are consumed.
 
 ### Task 1: Manage how AMIs are tested and rolled out
 
 Instead of just avoiding AMI upgrades, you can set up test clusters where you can try out new AMI releases before they are put into production.
 For example, you could have:
 
-* **Test clusters**: On these private clusters, you can run the latest AMIs for your workloads in a safe environment.
-* **Production clusters**: When you feel that everything is working properly, you can set the latest AMIs to be deployed in your production clusters so they are note upgraded.
-
-Remember that it is still best practice to gradually roll new AMIs into your cluster, even if they have been tested.
+* **Test clusters**: On these clusters, you can run the latest AMIs for your workloads in a safe environment. The `EC2NodeClass` for these clusters could be set with a chosen `amiFamily`, but no `amiSelectorTerms` set. For example, the `NodePool` and `EC2NodeClass` could begin with the following:
+  ```bash
+  apiVersion: karpenter.sh/v1beta1
+  kind: NodePool
+  metadata:
+    name: default
+  spec:
+    template:
+      spec:
+        nodeClassRef:
+          apiVersion: karpenter.k8s.aws/v1beta1
+          kind: EC2NodeClass
+          name: default
+  ---
+  apiVersion: karpenter.k8s.aws/v1beta1
+  kind: EC2NodeClass
+  metadata:
+    name: default
+  spec:
+    # The latest AMI in this family will be used
+    amiFamily: AL2
+  ```
+* **Production clusters**: When you feel that everything is working properly, you can set the latest AMIs to be deployed in your production clusters so they are not upgraded. One way to do that is to use `amiSelectorTerms` to set the tested AMI to be used in your production cluster. Refer to Task 2 for how to choose a particular AMI by `name` or `id`. Remember that it is still best practice to gradually roll new AMIs into your cluster, even if they have been tested. So consider implementing that for your production clusters as described in Task 3.
```

Comment:
The first sentence is kind of confusing to me. Is this what you meant?
```suggestion
* **Production clusters**: After you've confirmed that the AMI works in your lower environments, you can pin the latest AMIs to be deployed in your production clusters to roll out the AMI. One way to do that is to use `amiSelectorTerms` to set the tested AMI to be used in your production cluster. Refer to Task 2 for how to choose a particular AMI by `name` or `id`. Remember that it is still best practice to gradually roll new AMIs into your cluster, even if they have been tested. So consider implementing that for your production clusters as described in Task 3.
```

Reply:
Done.